book.json

```diff
@@ -1,5 +1,5 @@
 {
-  "gitbook": "3.2.2",
+  "gitbook": "3.2.3",
```

Comment:
this was needed to get the docs working, looks like they updated the latest version.